indexing/src/main/java/com/google/enterprise/cloudsearch/sdk/indexing/StructuredData.java

```diff
@@ -129,6 +131,7 @@
 
   private static final AtomicBoolean initialized = new AtomicBoolean();
   private static final List<DateTimeParser> dateTimeParsers = new ArrayList<>();
+  private static boolean ignoreConversionErrors = false;
```

Comment:
make this AtomicBoolean

Reply:
Why would it be required here ? The method where the Boolean is set is synchronized and protected by the configuration initialized atomic boolean.

Reply:
I went ahead and made it AtomicBoolean, just in case it is set outside of the init in the future